ipinfo/cmd_default.go

```diff
@@ -90,7 +115,8 @@ func cmdDefault() (err error) {
 	pflag.StringVarP(&fTok, "token", "t", "", "the token to use.")
 	pflag.BoolVar(&fNoCache, "nocache", false, "disable the cache.")
 	pflag.BoolVarP(&fVsn, "version", "v", false, "print binary release number.")
-	pflag.BoolVarP(&fHelp, "help", "h", false, "show help.")
+	pflag.BoolVarP(&fHelp, "short-help", "h", false, "show help.")
```

Comment:
Does it work if we remove the long-hand form? (i.e. `--short-help`, just make it empty string?)